scss/_custom-forms.scss

```diff
@@ -12,11 +12,10 @@
 .custom-control {
   position: relative;
   display: inline;
-  padding-left: 1.5rem;
-  color: #555;
```

Comment:
Ah, yeah, since we don't set a custom color for `.control-label`, I nuked this one, too.